src/__mocks__/geo.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
This is a way to mock user modules in jest - https://jestjs.io/docs/en/manual-mocks#mocking-user-modules
Required for mocking this in tests of `RequestService`. See line 3 of test/service/request-service.test.js